libr/util/assert.c

```diff
@@ -0,0 +1,25 @@
+#include <r_util.h>
+
+#define S_SIZE 200
+
+R_API void r_log_warn(const char *file, int line, const char *func, const char *warnexpr) {
+	char s[S_SIZE];
```

Comment:
why not use sdb_fmt

Reply:
it is possible but i don't see the advantage apart from depending on sdb.

Reply:
This is a layering issue. I think it is favorable to avoid the sdb dependency.